subprojects/zlib-ng.wrap

```diff
@@ -0,0 +1,9 @@
+[wrap-file]
+method = cmake
```

Comment:
no

Reply:
Can you be a bit more descriptive? Thanks

Reply:
The purpose of this repo is to provide meson build definitions.

Reply:
Hmm, I'm pretty fine with not having zlib-ng in wrap, thou I would at least be happy about a bit tech-support. Why does the cmake method make Mesa uncompilable, when I install zlib-ng system wide, it's completly fine?

Reply:
those errors come from GNU symbol versioning which is used in the codebase apparently.